### DISCUSSION TITLE
How can I get a github written permission to use the logo for sign up button?

### DISCUSSION THREAD

**Discussion by synestez** [community/community#86674] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I get a github written permission to use the logo for sign up button?
I'm using github to sign up to my website and it's written I need a permission, how can I obtain it?
Thanks in advance,

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @synestez ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
